### ISSUE TITLE
suggest: hashrate alarm

### ISSUE THREAD

**Issue by gvgit** [drpout/boilr#289]
In the long run, the price follows hashrate. For that I want suggest a hashrate alarm.

**Comment by dllud** [comment]
Think twice. Isn't it the other way around?

**Comment by gvgit** [comment]
No, it's trust which leads to investments. Nobody knew about iPhones 20
years ago, but many people wanted to have.

Check Max Keiser quotes about hashrate. He is right.

[ send by mobile device ]

dllud <notifications@github.com> schrieb am Mi., 25. Juli 2018, 21:36:
